clang/unittests/Parse/ParseHLSLRootSignatureTest.cpp

```diff
@@ -127,6 +127,7 @@ TEST_F(ParseHLSLRootSignatureTest, ValidParseEmptyTest) {
 }
 
 TEST_F(ParseHLSLRootSignatureTest, ValidParseDTClausesTest) {
+  using FlagT = llvm::dxbc::DescriptorRangeFlags;
```

Comment:
I wouldn't rename this to `FlagT`, but instead just introduce the `DescriptorRangeFlags` name to the scope. That is:
```suggestion
  using llvm::dxbc::DescriptorRangeFlags;
```

Then we can use `DescriptorRangeFlags::DataStaticWhileSetAtExecute` and the like without explicitly writing out the `llvm::dxbc::` namespace every time, which improves readability and in this case actually simplifies the diff quite a bit.